package.json

```diff
@@ -30,14 +30,14 @@
     "@lwc/synthetic-shadow": "1.0.2-222.23",
     "@lwc/wire-service": "1.0.2-222.23",
     "@salesforce/wire-service-jest-util": "^2.2.5",
-    "chalk": "^2.3.0",
-    "glob": "^7.1.2",
-    "jest": "24.8.0",
-    "yargs": "^13.0.0"
+    "chalk": "^2.4.2",
+    "glob": "^7.1.5",
+    "jest": "24.9.0",
```

Comment:
The main issue with a strict version match with `jest` is that the consumers won't get the benefits of the patches. What about using a `~` match for `jest`'s version?